src/main/webapp/js/chart-loader.js

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
I have lines 2 - 7 outside because we only need to call line 2 and 3 once. Let me know if I'm wrong, but I think we are fine with having them in the global scope.

Reply:
nvm, I'm dumb.

Reply:
You are smart. Everyone gets confused. There is no need to disparage oneself.